checks.d/docker_daemon.py

```diff
@@ -249,6 +254,21 @@ def check(self, instance):
         if self.collect_container_size:
             self._report_container_size(containers_by_id)
 
+        if self.collect_container_count:
+            self._report_container_count_by_state(containers_by_id)
+
+        if self.collect_dead_container_count:
```

Comment:
Would it be better to just report the number of containers by state for all states ? It could even be enabled by default as:
 - I believe the number of different possible states is bounded and small
 - We've already collected all the needed information (in `containers_by_id`)